torchvision/models/detection/transform.py

```diff
@@ -117,16 +126,19 @@ def forward(self,
                 targets[i] = target_index
 
         image_sizes = [img.shape[-2:] for img in images]
-        images = self.batch_images(images, size_divisible=self.size_divisible)
+        batched_images = self.batch_images(images, size_divisible=self.size_divisible)
```

Comment:
instead of 
```
images = self.batch_images(images, size_divisible=self.size_divisible)
```
I did
```
batched_images = self.batch_images(images, size_divisible=self.size_divisible)
```

This avoided the mypy error as it now is able to guess the type.